ui/src/components/system/TLSPolicy.vue

```diff
@@ -149,6 +149,9 @@ export default {
           context.TLSPolicy.policies = context.TLSPolicy.policies.concat(
             success.status.available
           );
+          // Sort the array by numerical ascending order
+          context.TLSPolicy.policies.sort((a,b) => a-b);
+          
```

Comment:
I don't get why we need to sort the array. The items order is already hardcoded here: https://github.com/NethServer/nethserver-cockpit/pull/264/files#diff-0993a3b6f24ef6461931c72ef20d0418R31

I'd prefer to fix it (if necessary) in that API `read` script.

```suggestion
```

Reply:
when I tested my code I saw that the array is not in the good order, I can let is as is or try to fix it. I suppose that javascript and bash did not agree on the type of variable. My fix is probably not the best but we have to fix something.

Reply:
yep it should be fixed, I put my glasses on my nose